lib/android.js

```diff
@@ -726,15 +726,14 @@ function findSDK(dir, config, androidPackageJson, callback) {
 		return callback(true);
 	}
 
-	const proguardPath = path.join(dir, 'tools', 'proguard', 'lib', 'proguard.jar'),
-		emulatorPath = path.join(dir, 'emulator', 'emulator' + exe),
+	const emulatorPath = path.join(dir, 'emulator', 'emulator' + exe),
 		result = {
 			path: dir,
 			executables: {
 				adb:       path.join(dir, 'platform-tools', 'adb' + exe),
-				emulator:  fs.existsSync(emulatorPath) ? emulatorPath : path.join(dir, 'tools', 'emulator' + exe)
+				emulator:  fs.existsSync(emulatorPath) ? emulatorPath : path.join(dir, 'emulator', 'emulator' + exe)
```

Comment:
No sense setting the emulator path to the same value when it doesn't exist. `emulatorPath` should always exist, right? If so, then just:

```
emulator: emulatorPath
```

 If not sure, then check if the file exists or set to `undefined`:

```
emulator:  fs.existsSync(emulatorPath) ? emulatorPath : undefined
```

Reply:
you are right :smile: make no sense that way. I've used your 2nd example but set it to `null` since the other values are `null` too